Makefile

```diff
@@ -40,6 +40,7 @@ DEFAULT_CFLAGS := \
     -Wshadow \
     -Wsign-compare \
     -Wstrict-prototypes \
+    -I./src/ \
```

Comment:
I don't think this is necessary.  (And if it were, it would go in `LOCAL_CFLAGS`, since `DEFAULT_CFLAGS` can be overridden.)